src/pages/Manager.vue

```diff
@@ -356,13 +367,15 @@ import GameRunnerProvider from '../providers/generic/game/GameRunnerProvider';
 import LocalFileImportModal from '../components/importing/LocalFileImportModal.vue';
 import { PackageLoader } from '../model/installing/PackageLoader';
 import GameInstructions from '../r2mm/launching/instructions/GameInstructions';
+import ServerListProvider from "../providers/components/loaders/ServerListProvider";
 
 @Component({
 		components: {
             LocalFileImportModal,
             OnlineModList: OnlineModListProvider.provider,
             LocalModList: LocalModListProvider.provider,
             NavigationMenu: NavigationMenuProvider.provider,
+            'serverlist': ServerListProvider.provider,
```

Comment:
Why is the key a string here? Can we use the same formatting as the others (consts?)?

Reply:
Also PascalCase since it's used with the other keys.